sdk/go/common/workspace/plugins.go

```diff
@@ -1513,6 +1513,20 @@ func IsPluginKind(k string) bool {
 	}
 }
 
+// IsPreinstalledLanguagePlugin returns true if the spec describes
+// a language plugin that comes preinstalled with the CLI.
```

Comment:
```suggestion
// a plugin that comes preinstalled with the CLI.
```
Not just language plugins

Reply:
The whole comment is wrong lol